src/ucs/sys/event_set.h

```diff
@@ -88,20 +93,19 @@ ucs_status_t ucs_event_set_del(ucs_sys_event_set_t *event_set, int event_fd);
  * Wait for an I/O events
  *
  * @param [in]  event_set          Event set created by ucs_event_set_create.
- * @param [in]  max_events         Maximum wait events.
+ * @param [in/out]  num_events     Number of expected/read events.
  * @param [in]  timeout_ms         Timeout period in ms.
  * @param [in]  event_set_handler  Callback functions.
  * @param [in]  arg                User data variables.
- * @param [out] read_events        Number of read events.
```

Comment:
nit. Do we need a variable alignment?

```
max_events
    num_events
```

To

```
max_events
num_events
```

Reply:
yes, missed this
fixed